partiql-ast/Cargo.toml

```diff
@@ -13,15 +13,15 @@ exclude = [
   "**/.github/**",
 ]
 edition = "2021"
-version = "0.1.0"
+version = "0.1.1-alpha.0"
```

Comment:
I'm curious is this the normal approach for Rust releases (i.e. to include the `-alpha` marker before the next release)? I saw that `ion-rust` and `ion-schema-rust` don't use this procedure

Reply:
I don't think we can pinpoint a general/normal approach necessarily for Rust as it's more related to the release lifecycle (not entirely certain though). On the other hand, knowing that using `cargo release` automatically adds the suffix in each post-release and in the next release gets removed automatically by the tool, I see this being beneficial for now. I also don't see this as an irreversible change since we're still at the early stages. This is what the tool does:

```bash
1. remove the suffix and publish—e.g. `0.1.1-alpha.0` -> `0.1.1`
2. bump the version w/ the new suffix `0.1.1` -> `0.1.2-alpha.0`
```